api/v1_users_coins.go

```diff
@@ -112,8 +135,13 @@ func (app *ApiServer) v1UsersCoins(c *fiber.Ctx) error {
 			balances_with_prices.balance_usd
 		FROM balances_with_prices
 		ORDER BY
+			-- Always show user's owned coins first, regardless of balance
+			balances_with_prices.is_owned_coin DESC,
+			-- Then prioritize AUDIO
 			balances_with_prices.ticker = '$AUDIO' DESC,
-			balances_with_prices.balance_usd DESC,
+			-- Then by number of coins (balance)
+			balances_with_prices.balance DESC,
+			-- Finally by mint for consistent ordering
```

Comment:
oof I sure hope we don't ever decide to add query param sorting to this endpoint... 😅